app/src/main/res/layout/fragment_word_detail.xml

```diff
@@ -89,36 +88,50 @@
                     android:layout_marginTop="@dimen/dp_10"
                     android:orientation="horizontal"
                     >
-                    <ImageView
-                        android:layout_width="@dimen/dp_20"
-                        android:layout_height="@dimen/dp_20"
-                        android:src="@drawable/ic_speaker"
-                        />
-                    <TextView
+                    <LinearLayout
                         android:layout_width="wrap_content"
                         android:layout_height="wrap_content"
-                        android:layout_gravity="center_vertical"
-                        android:text="@string/label_lang_uk"
-                        android:textColor="@color/color_black"
-                        android:textSize="@dimen/sp_15"
-                        android:textStyle="bold"
-                        />
-                    <ImageView
-                        android:layout_width="@dimen/dp_20"
-                        android:layout_height="@dimen/dp_20"
-                        android:layout_marginLeft="@dimen/dp_10"
-                        android:layout_marginStart="@dimen/dp_10"
-                        android:src="@drawable/ic_speaker"
-                        />
-                    <TextView
+                        android:onClick="@{(v) -> viewModel.speakUK(viewModel.word.word)}"
+                        android:orientation="horizontal"
+                        >
+                        <ImageView
+                            android:layout_width="@dimen/dp_20"
+                            android:layout_height="@dimen/dp_20"
+                            android:src="@drawable/ic_speaker"
+                            />
+                        <TextView
+                            android:layout_width="wrap_content"
+                            android:layout_height="wrap_content"
+                            android:layout_gravity="center_vertical"
+                            android:text="@string/label_lang_uk"
+                            android:textColor="@color/color_black"
+                            android:textSize="@dimen/sp_15"
+                            android:textStyle="bold"
+                            />
+                    </LinearLayout>
+                    <LinearLayout
```

Comment:
WARNING: [UseCompoundDrawables] This tag and its children can be replaced by one `<TextView/>` and a compound drawable
 A `LinearLayout` which contains an `ImageView` and a `TextView` can be more efficiently handled as a compound drawable (a single TextView, using the `drawableTop`, `drawableLeft`, `drawableRight` and/or `drawableBottom` attributes to draw one or more images adjacent to the text).

If the two widgets are offset from each other with margins, this can be replaced with a `drawablePadding` attribute.

There's a lint quickfix to perform this conversion in the Eclipse plugin.